src/lib/mapveu/analysis/MapAnalysis.tsx

```diff
@@ -0,0 +1,517 @@
+    filters: analysisState.analysis?.descriptor.subset.descriptor,
+    xAxisVariable: selectedVariables.overlay,
+    computationType: 'pass',
+    markerType: 'pie',
+    checkedLegendItems: undefined,
+    //TO DO: maybe dependentAxisLogScale
+  });
+
+  const finalMarkers = useMemo(() => markers || [], [markers]);
+
+  const dataClient = useDataClient();
+
+  const appPromiseState = usePromise(
+    useCallback(async () => {
+      const { apps } = await dataClient.getApps();
+      const app = apps.find((a) => a.name === 'pass');
+      if (app == null) throw new Error('Could not find pass app.');
+      return app;
+    }, [dataClient])
+  );
+
+  const computation = analysisState.analysis?.descriptor.computations[0];
+
+  const updateVisualizations = useCallback(
+    (
+      visualizations:
+        | Visualization[]
+        | ((visualizations: Visualization[]) => Visualization[])
+    ) => {
+      analysisState.setComputations((computations) =>
+        computations.map((c) =>
+          c.computationId !== computation?.computationId
+            ? c
+            : {
+                ...c,
+                visualizations:
+                  typeof visualizations === 'function'
+                    ? visualizations(c.visualizations)
+                    : visualizations,
+              }
+        )
+      );
+    },
+    [analysisState, computation?.computationId]
+  );
+
+  const onVisualizationCreated = useCallback(
+    (visualizationId: string) => {
+      setIsVizSelectorVisible(false);
+      setActiveVisualizationId(visualizationId);
+    },
+    [setActiveVisualizationId, setIsVizSelectorVisible]
+  );
+
+  const activeViz = analysisState.analysis?.descriptor.computations
+    .flatMap((c) => c.visualizations)
+    .find((v) => v.visualizationId === appState.activeVisualizationId);
+
+  const totalCounts = useEntityCounts();
+  const filteredCounts = useEntityCounts(
+    analysisState.analysis?.descriptor.subset.descriptor
+  );
+
+  const fieldTree = useFieldTree(
+    useFlattenedFields(studyEntities, 'variableTree')
+  );
+  const featuredFields = useFeaturedFields(studyEntities, 'variableTree');
+
+  const subsetVariableAndEntity = useMemo(() => {
+    if (appState.subsetVariableAndEntity)
+      return appState.subsetVariableAndEntity;
+    if (featuredFields.length) {
+      const [entityId, variableId] = featuredFields[0].term.split('/');
+      return { entityId, variableId };
+    } else {
+      const variable = findFirstVariable(
+        fieldTree,
+        studyMetadata.rootEntity.id
+      );
+      const [entityId, variableId] = variable?.field.term.split('/') ?? [];
+      return { entityId, variableId };
+    }
+  }, [
+    appState.subsetVariableAndEntity,
+    featuredFields,
+    fieldTree,
+    studyMetadata.rootEntity.id,
+  ]);
+
+  const fullScreenActions = (
+    <>
+      <div>
+        <Tooltip title="Delete visualization">
+          <button
+            type="button"
```

Comment:
```suggestion
            aria-label={`Delete ${activeViz?.displayName || "visualization."}`}
            type="button"
```

Reply:
This was from the github.dev tool, so apologies if this looks weird. Trying out the new tool :D